src/main/java/hibernate/service/impl/AuthentificationServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package hibernate.service.impl;
+
+import hibernate.exception.AuthenticationException;
+import hibernate.lib.Inject;
+import hibernate.lib.Service;
+import hibernate.model.User;
+import hibernate.service.AuthenticationService;
+import hibernate.service.UserService;
+import hibernate.util.SecurityUtil;
+import java.util.Optional;
+
+@Service
+public class AuthentificationServiceImpl implements AuthenticationService {
+    @Inject
+    private UserService userService;
+
+    @Override
+    public User login(String email, String password) throws AuthenticationException {
+        Optional<User> dirty = userService.findByEmail(email);
+        if (dirty.isEmpty()) {
+            throw new AuthenticationException("User doesn't exist");
```

Comment:
```suggestion
            throw new AuthenticationException("User doesn't exist");
```
try to use just one if block and one exception